### ISSUE TITLE
runnable PATH determination seems faulty

### ISSUE THREAD

**Issue by schmittberger** [PhilipSkinner/windows#1]
Hi, 
your runnable.js code to determine a bin containing PATH entry seems to have incorrect syntax.
It now is:
```
if (!exists(PATH)) {
  if (process.env.NODE_PATH) PATH = process.env.NODE_PATH
} else {
  throw new Error('Could not find a good path to install to');
}
```

I think it should be:
```
if (!exists(PATH)) {
  if (process.env.NODE_PATH) {
    PATH = process.env.NODE_PATH
  } else {
    throw new Error('No bin-PATH found');
  }
}
```

Otherwise I can't start my electron-app with windows required in it.

**Comment by Euclidite** [comment]
I've experienced this same issue - thanks for the suggested fix @schmittberger!

**Comment by icharge** [comment]
When `PATH` contain 1 path of Node it will throw like this. Same to me.

**Comment by PhilipSkinner** [comment]
Apologies for the delay, I've released version 0.1.1 that should have fixed this issue.
